spec/models/appointment_spec.rb

```diff
@@ -46,6 +46,17 @@
         expect(Appointment.overdue).not_to include(upcoming_appointment)
       end
     end
+
+    describe '.eligible_for_reminders' do
+      let!(:dead_patient) { create(:patient, status: 'dead') }
+      let!(:patient_with_consent_denied) { create(:patient, :denied) }
+      let!(:overdue_appointment_for_dead_patient) { create(:appointment, :overdue, patient: dead_patient) }
+      let!(:overdue_appointment_for_denied_patient) { create(:appointment, :overdue, patient: patient_with_consent_denied) }
```

Comment:
Let's explicitly schedule these appointments for 3 days ago. It's a lot clearer than inferring that the current default in the factory is for 3 days.